wordpress-mobile/WordPress-iOS
# Readability Part 2, life without WPTableViewCell (almost)

Rebuilt from: https://github.com/wordpress-mobile/WordPress-iOS/pull/5761

Part II of readability support actually starts to follow readable margins and content guides.

For the most part, we're really only tackling views and controllers that can support readability without many changes. We have cleaned up a few spots along the way, with a bit of use of `UIStackView`. Other views and controllers use the new `WPTableVieCell` toggle of `forceCustomCellMargins` to temporarily restrict the layout until those can be properly updated.

For this PR, note:

1. First requires reviewing, merging, and updating to WP-iOS-Shared pod `0.6.0` via PR: https://github.com/wordpress-mobile/WordPress-Shared-iOS/pull/105.
2. Merging this PR should happen alongside merging https://github.com/wordpress-mobile/WordPress-iOS/pull/5756.

To test:

(1.) Ensure the following views/controllers, on iPad, are following the default readable margins in both landscape and portrait orientation:

- Blog list
- Blog Details
  - Plans
  - Comments
  - Sharing
  - Settings
- Me tab
  - My Profile
  - Account Settings
  - App Settings
  - Notification Settings
  - Help & Support
- Reader tab
  - Followed Sites "Manage"
- Post Settings
  - Categories (with parent/child indentation)
  - Featured image
  - Location

(2.) Ensure the following views/controller are using the original margin hack (until they can be adapted):

- Notifications
- CommentViewController (when viewing a single comment)
- Pages
- Blog Posts
- People

Note: This PR will build with mis-aligned header views which are instead resolved via part 1: https://github.com/wordpress-mobile/WordPress-iOS/pull/5756
Note: Reader posts/cards are untouched for this go around.

Needs review: @frosty ready for your inspection sir 😁 👀
